conformance_tests/sysman/test_sysman_init/src/test_init_sysman.cpp

```diff
@@ -16,6 +16,226 @@
 
 namespace {
 
+std::vector<ze_driver_handle_t> get_ze_drivers() {
+  uint32_t ze_driver_count = 0;
+  EXPECT_EQ(ZE_RESULT_SUCCESS, zeDriverGet(&ze_driver_count, nullptr));
+  EXPECT_GT(ze_driver_count, 0);
+  std::vector<ze_driver_handle_t> ze_drivers(ze_driver_count);
+  EXPECT_EQ(ZE_RESULT_SUCCESS,
+            zeDriverGet(&ze_driver_count, ze_drivers.data()));
+  return ze_drivers;
+}
+
+std::vector<ze_device_handle_t> get_ze_devices(ze_driver_handle_t ze_driver) {
+  uint32_t ze_device_count = 0;
+  EXPECT_EQ(ZE_RESULT_SUCCESS,
+            zeDeviceGet(ze_driver, &ze_device_count, nullptr));
+  EXPECT_GT(ze_device_count, 0);
+  std::vector<ze_device_handle_t> ze_devices(ze_device_count);
+  EXPECT_EQ(ZE_RESULT_SUCCESS,
+            zeDeviceGet(ze_driver, &ze_device_count, ze_devices.data()));
+  return ze_devices;
+}
+
+std::vector<zes_driver_handle_t> get_zes_drivers() {
+  uint32_t zes_driver_count = 0;
+  EXPECT_EQ(ZE_RESULT_SUCCESS, zesDriverGet(&zes_driver_count, nullptr));
+  EXPECT_GT(zes_driver_count, 0);
+  std::vector<zes_driver_handle_t> zes_drivers(zes_driver_count);
+  EXPECT_EQ(ZE_RESULT_SUCCESS,
+            zesDriverGet(&zes_driver_count, zes_drivers.data()));
+  return zes_drivers;
+}
+
+std::vector<zes_device_handle_t>
+get_zes_devices(zes_driver_handle_t zes_driver) {
+  uint32_t zes_device_count = 0;
+  EXPECT_EQ(ZE_RESULT_SUCCESS,
+            zesDeviceGet(zes_driver, &zes_device_count, nullptr));
+  EXPECT_GT(zes_device_count, 0);
+  std::vector<zes_device_handle_t> zes_devices(zes_device_count);
+  EXPECT_EQ(ZE_RESULT_SUCCESS,
+            zesDeviceGet(zes_driver, &zes_device_count, zes_devices.data()));
+  return zes_devices;
+}
+
+void validate_sysman_api(zes_device_handle_t zes_device,
+                         ze_result_t expected_result) {
+  uint32_t count = 0;
+  ze_bool_t bool_false{};
+  zes_device_handle_t device_handle{};
+  zes_uuid_t uuid{};
+  zes_device_properties_t properties{};
+  zes_device_state_t state{};
+  zes_pci_properties_t pci_properties{};
+  zes_pci_state_t pci_state{};
+  zes_pci_stats_t pci_stats{};
+  zes_overclock_mode_t overclock_mode = ZES_OVERCLOCK_MODE_MODE_OFF;
+  zes_pending_action_t pending_action = ZES_PENDING_ACTION_PENDING_NONE;
+  zes_pwr_handle_t power_handle{};
+  zes_device_ecc_properties_t ecc_properties{};
+  const zes_device_ecc_desc_t ecc_state{};
+  zes_fabric_port_handle_t fabric_port{};
+  zes_fabric_port_throughput_t *fabric_throughput{};
+  std::vector<zes_event_type_flags_t> event_type{};
+  ze_result_t result{};
+
+  result = zesDeviceGetProperties(zes_device, &properties);
+  EXPECT_TRUE(result == expected_result ||
```

Comment:
The function is added to check whether sysman api's exit without any crash when core handles are passed . the reason why 2 expectation is set because some of the api's may be unsupported. unsupported api's may be different for different platforms.

hence if the checking whether the result is expected result or Unsupported error.